skimage/morphology/misc.py

```diff
@@ -124,3 +127,81 @@ def remove_small_objects(ar, min_size=64, connectivity=1, in_place=False):
     out[too_small_mask] = 0
 
     return out
+
+def remove_small_holes(ar, min_size=64, connectivity=1, in_place=False):
+    """Remove connected components smaller than the specified size within a 
+       larger connected object.
```

Comment:
+1 for "remove contiguous holes". Or just "remove holes"? I think it's clear that a hole is contiguous.